src/cljs/aikakonematka/web_socket.cljs

```diff
@@ -66,6 +72,10 @@
 
       :aikakone/reset (util/reset-game!)
 
+      :aikakone/music (when (util/currently-playing-game?)
+                        (sound/make-melody event-data)
```

Comment:
(sound/make-melody event-data) is doing nothing as it does not have side-effect or return value is not used anywhere.